### ISSUE TITLE
y-axis label erro in sumo-dosplot --subplot

### ISSUE THREAD

**Issue by ekkowdg** [SMTG-Bham/sumo#250]
When I am plotting the subplots, y-axis label is shown inside the plot (bottom one). Can you please tell how to fix this one?

**Comment by ajjackson** [comment]
@utf I can reproduce this with `sumo-dosplot --subplot` on the Cs2SnBr6 data; it looks different from the example in https://smtg-bham.github.io/sumo/gallery.html.

The x-coordinate placement is a bit odd, but stranger to me is that the text has an incorrect vertical placement: it looks like it is being placed relative to the axis rather than the figure.

![dos](https://github.com/user-attachments/assets/582d0525-ab09-4dae-8623-3cbc4d13cb4c)

**Comment by ekkowdg** [comment]
Dear ajjackson ,I think sumo is a very good and convenient tool for electronic structure analysis and processing, and I want to use it in my future papers, so I tried to uninstall and reload it, but it doesn't seem to solve the problem, and I am not the only one who has encountered this problem. It seems that there is a problem with the code that sets the Y-axis label when calling matplotlib to draw subgraphs

**Comment by ajjackson** [comment]
Right, I just said I can reproduce the behaviour. The next step is to figure out whether it is

- truly unintended behaviour,
- has changed over time,
- and if so, why it changed.

Otherwise we end up with a fix that only works on some machines or for some users!
